lib/contexts/DiceContext/DiceContext.tsx

```diff
@@ -26,18 +24,18 @@ export const DefaultDiceCommandOptions: Array<IDiceCommandOption> = [
 ];
 
 export function useDice() {
-  const [latestCommandOptions, setLatestCommandOptions] = useState<
-    Array<IDiceCommandOption>
-  >(DefaultDiceCommandOptions);
   const [options, setOptions] = useState<IRollDiceOptions>({
     listResults: false,
   });
 
   const [pool, setPool] = useState<IDicePool>([]);
   const [playerId, setPlayerId] = useState<string>();
-  const [commandGroups, setCommandGroups] = useState<Array<IDiceCommandGroup>>(
-    []
-  );
+  const [commandGroups, setCommandGroups] = useState<Array<IDiceCommandGroup>>([
+    AllDiceCommandGroups["4dF"],
+  ]);
+  const [commandGroupsBeforePool, setCommandGroupsBeforePool] = useState<
+    Array<IDiceCommandGroup>
+  >([]);
```

Comment:
To store commands before a pool is created so that we can reset them once the pool is rolled